samples/snippets/mf_explicit_model_test.py

```diff
@@ -0,0 +1,163 @@
+
+        ratings_csv = io.BytesIO(ratings_content.replace(b"::", b","))
+        ratings_config = google.cloud.bigquery.LoadJobConfig()
+        ratings_config.source_format = "CSV"
+        ratings_config.write_disposition = "WRITE_TRUNCATE"
+        ratings_config.schema = [
+            google.cloud.bigquery.SchemaField("user_id", "INT64"),
+            google.cloud.bigquery.SchemaField("item_id", "INT64"),
+            google.cloud.bigquery.SchemaField("rating", "FLOAT64"),
+            google.cloud.bigquery.SchemaField("timestamp", "TIMESTAMP"),
+        ]
+        bqclient.load_table_from_file(
+            ratings_csv, "bqml_tutorial.ratings", job_config=ratings_config
+        ).result()
+
+        # Upload the movie data into the movies table.
+        with ml1m_zip.open("ml-1m/movies.dat") as movies_file:
+            movies_content = movies_file.read()
+
+        movies_csv = io.BytesIO(movies_content.replace(b"::", b"@"))
+        movies_config = google.cloud.bigquery.LoadJobConfig()
+        movies_config.source_format = "CSV"
+        movies_config.field_delimiter = "@"
+        movies_config.write_disposition = "WRITE_TRUNCATE"
+        movies_config.schema = [
+            google.cloud.bigquery.SchemaField("movie_id", "INT64"),
+            google.cloud.bigquery.SchemaField("movie_title", "STRING"),
+            google.cloud.bigquery.SchemaField("genre", "STRING"),
+        ]
+        bqclient.load_table_from_file(
+            movies_csv, "bqml_tutorial.movies", job_config=movies_config
+        ).result()
+    # [END bigquery_dataframes_bqml_mf_explicit_upload_movielens]
+
+    # [START bigquery_dataframes_bqml_mf_explicit_create]
+    from bigframes.ml import decomposition
+    import bigframes.pandas as bpd
+
+    # Load data from BigQuery
+    bq_df = bpd.read_gbq(
+        "bqml_tutorial.ratings", columns=("user_id", "item_id", "rating")
+    )
+
+    # Create the Matrix Factorization model
+    model = decomposition.MatrixFactorization(
+        num_factors=34,
+        feedback_type="explicit",
+        user_col="user_id",
+        item_col="item_id",
+        rating_col="rating",
+        l2_reg=9.83,
+    )
+    model.fit(bq_df)
+    model.to_gbq(
+        your_model_id, replace=True  # For example: "bqml_tutorial.mf_explicit"
+    )
+    # [END bigquery_dataframes_bqml_mf_explicit_create]
+    # [START bigquery_dataframes_bqml_mf_explicit_evaluate]
+    # Evaluate the model using the score() function
+    model.score()
+    # Output:
+    # mean_absolute_error	mean_squared_error	mean_squared_log_error	median_absolute_error	r2_score	explained_variance
+    # 0.485403	                0.395052	        0.025515	            0.390573	        0.68343	        0.68343
+    # [END bigquery_dataframes_bqml_mf_explicit_evaluate]
+    # [START bigquery_dataframes_bqml_mf_explicit_recommend_df]
+    # Use predict() to get the predicted rating for each movie for 5 users
+    subset = bq_df.head(5)
```

Comment:
The SQL sample only includes the `user_id` column. Could we try something similar?

```suggestion
    subset = bq_df[["user_id"]].head(5)
```